File: parser-test/src/main/java/ru/spbau/kaysin/parser/domain/Class.java
package ru.spbau.kaysin.parser.domain;


import java.util.Collection;


/**
 * Class representing java class.
 */
public class Class {
    private final String name;
    private final Collection<Method> methods;

    public Class(String name, Collection<Method> methods) {
        this.name = name;
        this.methods = methods;
    }

    @Override
    public String toString() {
        return "Class{" +
            "\nname='" + name + '\'' +
            "\nmethods=" + methods +
            '}';
    }
}

File: parser-test/src/main/scala/ru/spbau/kaysin/scala/parser/domain/Class.scala
package ru.spbau.kaysin.scala.parser.domain

/**
  * Class representing java class.
  */
class Class(val name: String, val methods: Set[Method]) {
  override def toString: String = "Class{" + "\nname='" + name + '\'' + "\nmethods=" + methods + '}'
}